tests/e2e/tests/galley/testdata/mixer-handler.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "config.istio.io/v1alpha2"
```

Comment:
This adds burden to update tests files when config apis changes.
is there a way that we can reuse existing yamls to reduce test maintenance cost?

Reply:
We'll need valid and invalid variants of each group/version/kind. And I don't think we have that exhaustive list in istio/istio. I figured it was better to be consistent and define them in one testdata location. 

GVK shouldn't change that frequently either (post config group rename churn) and the total list of mixer kinds should be reduced by an order of magnitude once mixer config is refactored into common instance and handler kinds.